ddp-workspace/projects/ddp-dsm-ui/src/app/services/auth.service.ts

```diff
@@ -171,8 +171,7 @@ export class Auth implements OnDestroy {
   public buildHeaders(): any {
     const headers = new HttpHeaders({
       'Content-Type': 'application/json',
-      Accept: 'application/json',
```

Comment:
This is only called by code that hits the `ui/auth0` route, so it's safe to just remove the token from the header.